agent/lib_guzzle_common.c

```diff
@@ -272,19 +315,19 @@ char* nr_guzzle_response_get_header(const char* header,
 }
 
 NR_PHP_WRAPPER_START(nr_guzzle_client_construct) {
```

Comment:
Great point Michal! This is performing an unnecessary extra step because we have already identified the correct version by determining the correct file name in nr_execute_handle_library and can instead call the specific guzzle instrumentation in nr_guzzleX_enable

Reply:
I have removed this function and now directly call the correct guzzle instrumentation inside of nr_guzzle_enable! [6a879ff](https://github.com/newrelic/newrelic-php-agent/pull/498/commits/6a879ff10ae7093861b2b3f5893d112b00553284)